### PR TITLE
Minor: add another emoji, import a couple of changes for Enterprise

### PR DESCRIPTION
**Pull request by mikehearn**

### CODE REVIEW

File: gradle-plugins/cordformation/src/main/groovy/net/corda/plugins/Node.groovy [hunk 1]
```diff
@@ -218,7 +218,7 @@ class Node extends CordformNode {
      */
     private File verifyAndGetCordaJar() {
         def maybeCordaJAR = project.configurations.runtime.filter {
-            it.toString().contains("corda-${project.corda_release_version}.jar")
+            it.toString().contains("corda-${project.corda_release_version}.jar") || it.toString().contains("corda-enterprise-${project.corda_release_version}.jar")
```
Comment (shamsasari) [inline gradle-plugins/cordformation/src/main/groovy/net/corda/plugins/Node.groovy]: enterprise??

Comment (mikehearn) [inline gradle-plugins/cordformation/src/main/groovy/net/corda/plugins/Node.groovy]: Yes, it's deliberate. It's not a secret that there is an enterprise version of the product, and it's not worth having a diff here.

Comment (shamsasari) [inline gradle-plugins/cordformation/src/main/groovy/net/corda/plugins/Node.groovy]: Having a diff a name prefix constant would be better.

**Comment by mikehearn** [comment]
Yes but the idea is to eliminate diffs where possible, so moving it to a prefix variable wouldn't help here.

**Pull request merged**